metadata.yaml

```diff
@@ -17,4 +18,5 @@ containers:
 resources:
   jenkins-agent-image:
     type: oci-image
-    description: OCI image for jenkins-agent
\ No newline at end of file
+    description: OCI image for jenkins-agent
+    auto-fetch: true
```

Comment:
What does this do? I don't see it mentioned on https://juju.is/docs/sdk/metadata-reference

Reply:
I'm not sure, I just copied the indico template file and adjusted it for jenkins agent: https://github.com/canonical/indico-operator/blob/improve-docs/metadata.yaml. I found it referenced here, it seems to somehow speed up docker interactions although I didn't quite follow the explanation: https://github.com/juju-solutions/layer-docker-resource